users/views.py

```diff
@@ -0,0 +1,44 @@
+import json, re
+from django.http.response import JsonResponse
+
+from django.views import View
+from django.http  import JsonResponse
+from django.db    import IntegrityError
+
+from .models      import User
+
+class SignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            password_regx = re.compile('^.*(?=^.{10,15}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%^&+=]).*$')
+            email_regex   = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+
+            if User.objects.filter(email = data['email']).exists():
+                return JsonResponse({'message': 'EMAIL_EXIST'}, status=400)
+
+            if User.objects.filter(mobile = data['mobile']).exists():
+                return JsonResponse({'message': 'MOBILE_EXIST'}, status=400)
+
+            if not email_regex.match(data['email']):
+                return JsonResponse({'message':'PLEASE ENTER @ OR .'}, status=400)
```

Comment:
```suggestion
                return JsonResponse({'message':'INVALID_EMAIL'}, status=400)
```
INVALID_EMAIL로 통일하면 좋을 것 같습니다.(무조건 바꾸라는 것은 아니에요)

Reply:
통일하는게 좋을 것 같아 수정했습니다!